test/components/octicon_component_test.rb

```diff
@@ -59,9 +59,27 @@ def test_renders_class_passed_in
     assert_selector(".foo")
   end
 
+  def test_renders_classes_passed_in
+    render_inline(Primer::OcticonComponent.new(icon: "star", classes: "foo"))
+
+    assert_selector(".foo")
+  end
+
   def test_renders_class_passed_in_along_with_primer_class
     render_inline(Primer::OcticonComponent.new(icon: "star", class: "foo", my: 4))
 
     assert_selector(".foo.my-4")
   end
+
+  def test_renders_concatenated_class_and_classes_passed_in
+    render_inline(Primer::OcticonComponent.new(icon: "star", class: "foo", classes: "bar"))
+
+    assert_selector(".foo.bar")
+  end
+
+  def test_does_not_render_classify_attributes_as_html_attributes
+    render_inline(Primer::OcticonComponent.new(icon: "star", classes: "foo", my: 4, display: [:none]))
+
+    refute_selector('[classes="foo"][display="none"]')
```

Comment:
Might we simplify this test as follows:

```suggestion
    render_inline(Primer::OcticonComponent.new(icon: "star", classes: "foo"))

    refute_selector('[classes="foo"]')
```

Reply:
@joelhawksley the point of the test is so it doesn't put `display` attribute in the html test. I think having the two, classes, and display, reinforces the test.

Reply:
@heynan0 my concern with asserting both at the same time is that it makes `refute_selector` risky to use here. 

If we introduced a bug that say, didn't filter out `classes`, but did filter out `display`, this test would still pass, right?

Reply:
@joelhawksley makes sense. I split it into two statements.